packages/block-library/src/navigation-link/edit.js

```diff
@@ -114,6 +113,41 @@ function NavigationLinkEdit( {
 		selection.addRange( range );
 	}
 
+	async function handleCreatePage( pageTitle ) {
+		const type = 'page';
+		const page = await saveEntityRecord( 'postType', type, {
+			title: pageTitle,
+			status: 'publish',
+		} );
+
+		// `page` may not reject the Promise
+		// but may still be invalid. Here we
+		// tests for unexpected values and throw accordingly.
+		if ( null === page || undefined === page ) {
```

Comment:
It was noted in an earlier review: When will `page` be `null`?

Reply:
I believe I did respond in that review. I found when testing by blocking the network requests I received `null` as the value of `page. 

Open devtools -> Network -> Block request to "Create pages" endpoint -> Try to create a page -> see `null` response.

Reply:
![Screen Capture on 2020-02-17 at 11-25-56](https://user-images.githubusercontent.com/444434/74649910-74756f00-5178-11ea-87f6-ccc86a47813e.gif)